tests/impulse-tests/Makefile

```diff
@@ -362,13 +362,13 @@ interp1:
 #########################################################################
 # Rust backend
 rust:
-	$(MAKE) -f Make.rust FAUSTOPTIONS="-I dsp -double"
+	$(MAKE) -f Make.rust FAUSTOPTIONS="-I dsp -double -rnt"
```

Comment:
In my opinion the CI should always use the mode that generates code against the trait.

As explained in the old PR, this largely reduces the risk of making an accidental breaking change. If the CI no longer runs against the trait based generation it would become rather easy to open a PR and merge it, and by accident breaks the interface in a fundamental way.

Personally I would prefer if `-ec` and `-os` would also use _their own_ traits instead to clearly communicate what the stable interface of these modes is.

Reply:
Ok i will setup different combinations of flags for the tests. 

I don't want to work on faust generating traits now because i think traits are a separate problem to solve. People who want a simple way of using faust can just use less flags.